docs/generate-time-series/go.rst

```diff
@@ -239,25 +245,26 @@ Then, in your main function, connect to CrateDB using the `Postgres Wire Protoco
         } else {
             fmt.Println("CONNECT OK")
         }
-	    defer conn.Close(context.Background())
+        defer conn.Close(context.Background())
 
-        conn.Exec(context.Background(), "CREATE TABLE [ IF NOT EXISTS ] iss (
-                                            timestamp TIMESTAMP GENERATED ALWAYS AS CURRENT_TIMESTAMP, 
-                                            position GEO_POINT)")
+        conn.Exec(context.Background(),
```

Comment:
@infoverload reformatted for readability (i.e., this makes it so the reader doesn't have to scroll the code block horizontally)